src/components/common/ColonyActions/ColonyActions.tsx

```diff
@@ -180,7 +180,7 @@ const ColonyActions = (/* { ethDomainId }: Props */) => {
             colonyAddress: colony.colonyAddress,
             colonyName: colony.name,
             nativeTokenAddress: colony.nativeToken.tokenAddress,
-            amount: BigNumber.from(1),
+            amount: BigNumber.from(5).mul(BigNumber.from(10).pow(18)), // this is in wei
```

Comment:
Are these values hardcoded for testing purpouses, or it there something I don't understand ?

Reply:
Correct, we don't have the UAC (I think that's what it's called) ported yet. They are here to trigger the relevant actions for testing

Reply:
All good then :+1: